pipenv/core.py

```diff
@@ -168,7 +169,7 @@ def cleanup_virtualenv(bare=True):
         click.echo(crayons.red('Environment creation aborted.'))
     try:
         # Delete the virtualenv.
-        shutil.rmtree(project.virtualenv_location, ignore_errors=True)
+        rmtree(project.virtualenv_location)
```

Comment:
I did specifically remove `ignore_errors=True` because I would much rather get an exception than silently leave stuff behind that may cause trouble in the future.

I did the test below with the `pip-selfcheck.json` file open so it couldn't be deleted.

```
PS C:\epc\t\812\testy> ..\pipenv\venv\scripts\pipenv --rm
Removing virtualenv (C:\Users\sda\.virtualenvs\testy-7ppH9Nsf)…
[WinError 145] The directory is not empty: 'C:\\Users\\sda\\.virtualenvs\\testy-7ppH9Nsf'
```

Reply:
The thing is, if we don’t silent the errors, the tests won’t even pass consistently on Windows, because how file handles are leaked everywhere in Python. If you really want this information, print out a warning. Don’t let it fail here.

Reply:
It was already being caught and `echo()`ed but I'll push a version that I think will output a bit more inline with the rest of pipenv messaging.